Core/include/Acts/Vertexing/HelicalTrackLinearizer.ipp

```diff
@@ -190,11 +190,8 @@ Acts::Result<Acts::LinearizedTrack> Acts::
   BoundVector constTerm =
       paramsAtPCA - positionJacobian * pca - momentumJacobian * momentumAtPCA;
 
-  // Set the parameter weight matrix
-  ActsSymMatrix<5> parWeight = (parCovarianceAtPCA.block<5, 5>(0, 0)).inverse();
-
-  BoundSymMatrix weightAtPCA{BoundSymMatrix::Identity()};
-  weightAtPCA.block<5, 5>(0, 0) = parWeight;
+  // The parameter weight
+  BoundSymMatrix weightAtPCA = parCovarianceAtPCA.inverse();
```

Comment:
I agree - I would also prefer to only pass along the 5x5 covariance but I fear that is not an option anymore after time is everywhere. I think the `HelicalTrackLinearizer` should pass along the full covariance even if the jacobian is incomplete for now because just putting `var_t=1` also seems wrong to me.

But I am happy to wait for the complete jacobian